ui/api.py

```diff
@@ -87,6 +87,7 @@ def post_hls_to_edx(video_file):
     responses = {}
     for edx_endpoint in edx_endpoints:
         try:
+            edx_endpoint.refresh_access_token()
```

Comment:
This was throwing an error for me when testing against RC (403), but maybe I had the wrong .env values set up.  Still working on getting a local edx instance running.